app/models/apple/config.rb

```diff
@@ -20,6 +20,8 @@ class Config < ApplicationRecord
     delegate :public_feed, to: :podcast, allow_nil: true
     alias_method :private_feed, :feed
 
+    accepts_nested_attributes_for :key, allow_destroy: true, reject_if: :all_blank
```

Comment:
We shouldn't allow deleting these, so I'd remove that bit.

Also - you'll want to remove the `reject_if: :all_blank`.  That way, if you try to create one of these with all fields blank, it'll throw a validation error.